pkg/audit/verifier_test.go

```diff
@@ -523,3 +525,65 @@ func TestVerifierModifiedSegment(t *testing.T) {
 		assert.Empty(t, report)
 	})
 }
+
+func TestVerifierModifiedSegmentFailsOnce(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		err := planet.Satellites[0].Audit.Service.Close()
+		require.NoError(t, err)
+
+		ul := planet.Uplinks[0]
+		testData := testrand.Bytes(8 * memory.KiB)
+
+		err = ul.Upload(ctx, planet.Satellites[0], "testbucket", "test/path", testData)
+		require.NoError(t, err)
+
+		cursor := audit.NewCursor(planet.Satellites[0].Metainfo.Service)
```

Comment:
No need to create a new Cursor. Just use the existing one: `audits.Cursor.NextStripe(ctx)`